Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -833,14 +863,14 @@ namespace OpenInfraPlatform {
 					secondOrthogonalDirection.normalize();
 		
 					//Calculate distance of the plane to the origin
-					double distance = (normalVector.x*arcStart.x) + (normalVector.y*arcStart.y) + (normalVector.y*arcStart.y);
+					double distance = (normalVector.x*arcStart.x) + (normalVector.y*arcStart.y) + (normalVector.z*arcStart.z);

```

Comment:
This hsould probably also work as `normalVector * arcStart`

Reply:
Unfortunately, this doesn’t work

Reply:
Resolved: bb58436